app/lib/utils.dart

```diff
@@ -36,10 +37,14 @@ Future<List<String>> listTarball(String path) async {
 }
 
 Future<String> readTarballFile(String path, String name) async {
-  final result = await Process.run('tar', ['-O', '-xzf', path, name]);
+  final result = await Process.run(
+    'tar',
+    ['-O', '-xzf', path, name],
+    stdoutEncoding: null,
```

Comment:
Couldn't you just pass in `const UtfCodec.allowMalformed: true` to the `stdoutEncoding` param?